tools/update_setup_version_from_git.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
```

Comment:
It might be helpful to add a few comments here to tell what's the purpose of this script and what's the expected output.

Reply:
I added a comment.